website/routes.py

```diff
@@ -864,6 +864,16 @@ def make_url_map(app):
                 '/project/<pid>/node/<nid>/statistics/',
             ],
             'get',
+            project_views.node.project_statistics_redirect,
+            json_renderer
```

Comment:
This is a web route, so the OsfWebRenderer should be used; you can use `notemplate`, since the view just redirects.